util/src/main/java/org/folio/util/PercentCodec.java

```diff
@@ -101,7 +103,7 @@ public static Appendable encode(Appendable appendable, CharSequence charSequence
    * // yields "https://example.com/users?query=status%20%3D%3D%20open"
    * </pre>
    *
-   * @param charSequence the input in UTF8 encoding; if null or empty then "" is returned
+   * @param charSequence in UTF8 encoding; if null or empty then "" is returned
```

Comment:
The parameter is called `in` but you are referring to it as `charSequence`.

Reply:
Thanks, fixed.